8/README.md

```diff
@@ -0,0 +1,139 @@
+
+This Specification is a [free and open standard](http://www.digistan.org/open-standard:definition) and is governed by the Digital Standards Organization's [Consensus-Oriented Specification System](http://www.digistan.org/spec:1/COSS).
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+
+## Tango Device Server specification
+
+ The specication of '''Device Server''' aims the process of Device management, being it self a Device.
+
+### Goals
+
+ A Device Server in a Tango Controls System aims to control the process of communication and lifecycle of one or several Device.
+
+Additionally, it aims to:
+
+* Provide a blackbox that register the operation executed on every Device
+
+* Be usable as a Device
+
+### Use Cases
+
+There are X main use cases for Device Server:
+
+* to manage several Device in the same process
+
+* to share the same channel of communication
+
+* to serialise the operation per Device
+
+
+## Specification
+
+The Device Server manage the communication with one or several Device.  Additionally it enforces the behaviour compatibility with the Tango Control System and offer different meta service.
+
+The Device Server SHALL follow these specification below in order to  control and monitor this communication:
+
+* A Device Server represents/interfaces the process which manage one or several Device
+
+* A Device Server is a also a Device itself.
+
+* A Device MUST NOT be managed by more than one Device Server
+
+The name of the Device Server SHALL follow this convention:
+```ABNF
+device-server-name = dserver "/" server "/" instance
+dserver = "dserver" ; dserver is a reserved domain following the device-name convention.
+server = family
+instance = member
+```
+
+The specification introduces as well the definition of the process which are used for the Device Server Name:
+
+* A Server is a type of process managed by a Device Server
+
+* An Instance is an unique identifier as several processes of the same Server MAY exist in a Tango Control System
+
+* A Server Instance is finally what identify the actual process in the Tango Control Sytem
+
+* A Server Instance CAN be interfaced by only one Device Server
+
+
+Additionally:
+* A Device Server MAY be call DServer or Admin Device but the terminology Device Server is RECOMMENDED
```

Comment:
Well, Administration Device is quite common use, why do you suggest to drop it?

Reply:
DServer and Admin device  are used for the same meaning in different facilities. But for reference we can recommend one. I choose the one from the RFC title.

Reply:
Sounds good

Reply:
A device server MAY be called Admin Device?
I never heard this one.
For me, the admin device is a _Device_ (not a _Device Server_).
For me, an admin device has a device name like dserver/server_exec_name/instance_name

Reply:
@bourtemb good comment! Effectively Device Server are called from different name depending on the facility. But the good news is we are all right. In this RFC I wrote a Device Server is Tango Device it self and naming convention is dserver/server/instance

Reply:
OK, so we definitely are using different names here in different institutes.
For me a device server is something different than an admin device.
I would not write:
"A Device Server SHALL be a Tango Device itself"
because this will bring confusion and encourage users to use the words "Device Server" and "Tango Device" for the same thing. They are not the same things. A device server is exporting and managing some devices. It is not the same thing. The admin device is a special device which SHALL be exported by any Tango Device Server and which is there to provide an interface to interact with the Device Server.
Instead of "A Device Server SHALL be a Tango Device itself", I would rather write something like:
"A Device Server SHALL export at least an administration Device"

Reply:
This is very similar to what I wrote in the PR #79, last line:
- The Device Server SHALL have a management device
Do we want to allow for both the forms "administration" and "management" Device?

Reply:
I see your point. If I understood you, '''Device Server''' is the system process and the '''Admin Device''' is the Tango Device which connected to the process '''Device Server'''.
This is very confusing considering that the Class is DServer (BTW missing in the RFC).
For me it's OK to call '''Device Server''' either the process of managing device and the dserver/xxx/yyy Device. At the end this is the role of this Tango Device to manage other devices.

Reply:
I think the people were too lazy to call it the "Device Server Device" like we would say for the "PowerSupply Device"

Reply:
> I see your point. If I understood you, '''Device Server''' is the system process and the '''Admin Device''' is the Tango Device which connected to the process '''Device Server'''.

Yes.

> This is very confusing considering that the Class is DServer (BTW missing in the RFC).

I understand. So the confusion is coming from the admin device class name.
The admin device is a device of class DServer. It is an object representing/controlling the Device Server, so it was quite natural to name this class like that (or DeviceServer).
When you have a Tango class controlling a PowerSupply, you naturally name this class _PowerSupply_. Same principle, here.

> For me it's OK to call '''Device Server''' either the process of managing device and the dserver/xxx/yyy Device. 

I would personally try to avoid to call a Device, a Device Server. _C'est un abus de langage_, as we would say in French. We should not encourage misleading terms in the specifications I think.

> At the end this is the role of this Tango Device to manage other devices.

The role of the admin device is to manage the Device Server, so part of it is to manage other devices, yes.

Reply:
> This is very similar to what I wrote in the PR #79, last line:
> 
>     * The Device Server SHALL have a management device
> 
> 
> Do we want to allow for both the forms "administration" and "management" Device?

I guess what is important is the role of the device here. So we can put what we want.
But if we want to relate more easily to the current documentation and tools, I think it would be better to use the term _administration device_.

Reply:
> The role of the admin device is to manage the Device Server, so part of it is to manage other devices, yes.
Is it good to say :
 the Device Server for the role of managing devices,
 the Device Server SHALL be implemented as a Tango Device, which is called the Admin Device
 the Admin Device should follow this naming convention ... dserver/server/instance

Reply:
Maybe this could help:
Extract from the Tango-Controls documentation's glossary:

> **device server**
    A Device Server is a program (executable) which is able to create devices of certain classes. A Device Server may implement one or multiple classes and instantiate one or more devices. A running device server is called a Device Server instance.
>
> **instance**
**device server instance**
    A running device server is called a device server instance. So it means, it is a process. Every device server instance has an unique name in Tango Controls by which it can be referenced. The name is built as {DeviceServerName}/{instanceName}. For each running device server the system creates a special device of DServer class: dserver/{DeviceServerName}/{instanceName}. This device provides a management facility for the corresponding device server instance (see DServer class device commands).

This page might help as well?:
https://tango-controls.readthedocs.io/en/latest/development/advanced/TangoDeviceServerModel.html

Reply:
My comments:
I have always used the same definitions of the Tango documentation. So one device of class "DServer" is a device started automatically by the "Device Server". Maybe a specific RFC should be created for the "DServer"? But the terminology distinction between DServer and 'Device Server' have always been confusing at SOLEIL, especially for new comers.  So "AdminDevice" seems to be more explicit for me so I have used that name in JTango.

Reply:
This paragraph has been rewritten according to your comments. @bourtemb @lorenzopivetta @gwen-soleil Can you review this paragraph?